packages/compass-e2e-tests/helpers/commands/show-shell.ts

```diff
@@ -13,3 +13,12 @@ export async function showShell(browser: CompassBrowser): Promise<void> {
     await browser.clickVisible(Selectors.ShellInput);
   });
 }
+
+export async function executeInShell(
```

Comment:
Our convention is one helper command file per command, so you probably want to add execute-in-shell.ts next to show-shell.ts and add a line for it next to wherever show-shell.ts was added.

Reply:
We started off having all the commands in one file but that quickly grew out of control so Sergey helped me with some amazing typescript: https://github.com/mongodb-js/compass/blob/main/packages/compass-e2e-tests/helpers/compass-browser.ts#L1-L13